index.html

```diff
@@ -585,15 +585,15 @@ <h3>Use Cases and Requirements</h3>
 
       <section id="conformance" class="normative">
         <p>
-A <dfn>conforming document</dfn> is any concrete expression of the data model
-that complies with the normative statements in this specification.
-Specifically, all relevant normative statements in Sections
+A <dfn>conforming document</dfn> is a compact JSON-LD document that
+complies with all of the relevant "MUST" statements in this specification.
+Specifically, the relevant normative "MUST" statements in Sections
 <a href="#basic-concepts"></a>, <a href="#advanced-concepts"></a>, and
-<a href="#syntaxes"></a> of this document MUST be enforced. A serialization
-format for the conforming document MUST be deterministic, bi-directional,
-and lossless as described in Section <a href="#syntaxes"></a>.
-The <a>conforming document</a> MAY be transmitted or stored in any such
-serialization format.
+<a href="#syntaxes"></a> of this document MUST be enforced. Other serialization
+formats for the conforming document MAY be used, and if they are, further
+guidance in Section <a href="#ecosystem-compatibility"></a> MUST be
+followed. The <a>conforming document</a> MAY be transmitted
+or stored in any such serialization format.
```

Comment:
"Other serialization formats for the conforming document MAY be used" except that it's just said that "A <dfn>conforming document</dfn> is a compact JSON-LD document" which implies rather strongly that "A <dfn>conforming document</dfn> is" not in any "Other serialization formats".

How a reader is to determine which statements (normative or otherwise) are _relevant_ would be a challenge we're now laying out to them, rather than solving it ourselves, by making the relevance of such statements unquestionably obvious.

_NOTE: The suggested revision cannot be applied automatically, because it includes both of the line ranges shown above as inserts 588-591 and 592-596 (and below as deletions 588-591 and 597-601), as well as the line range shown both above and below as deletions 592-596._
```suggestion
A <dfn>conforming document</dfn> is, or can be deterministically transformed
into, a compact JSON-LD document that complies with all of the relevant
normative "MUST" statements in this specification. More specifically, the
document MUST comply with all relevant normative "MUST" statements in Sections
<a href="#basic-concepts"></a>, <a href="#advanced-concepts"></a>, and
<a href="#syntaxes"></a> of this document. If the document is serialized as
anything other than compact JSON-LD, further guidance in Section
<a href="#ecosystem-compatibility"></a> MUST be followed.
```

Reply:
@TallTed, I don't think we want to change the definition of the conforming document -- but we may want to say that the conforming document can be stored / transported / enveloped using other formats provided that the conforming document can be deterministically reproduced. I think that's the aim here.

Reply:
@msporny @dlongley 

The definition is here being changed from the previous text to the new, and in ways which seem (potentially) problematic to me. To start with, from --
> any concrete expression of the data model
that complies with the normative statements in this specification.

-- to --
> a compact JSON-LD document that
complies with all of the relevant "MUST" statements in this specification.

The first is VERY broad and open. The second is VERY narrow and closed.

As I said in my intro to the change suggested above, the new text starts by saying that only Compact (where I capitalize the `C` because it may easily be overlooked otherwise, suggesting that any form of JSON-LD is sufficient) JSON-LD _might_ (if it complies with all the MUSTs) be a conforming document.

Only later does the new text suggest that other formats *might* be usable, but only (it appears to me) for storage or transmission (so evaluation, verification, etc., first requires transformation to Compact JSON-LD), making me question what value there could be in such storage of transmission and further wonder why not simply require that it always be kept in Compact JSON-LD?

Reply:
The problematic text has been removed in commit (based on consensus reached in the last WG meeting): a515e6fbfed5b28bd31af56fac790836567b8242